src/lib.rs

```diff
@@ -447,6 +449,16 @@ mod contract {
         EthConnectorContract::get_instance().ft_on_transfer(&engine)
     }
 
+    #[no_mangle]
+    pub extern "C" fn get_paused_flags() {
+        EthConnectorContract::get_instance().get_paused_flags()
```

Comment:
As discussed elsewhere, I don't like this pattern of doing IO in business logic (outside `lib.rs`). It's fine to merge as is, but I'd like to avoid this in the future and go back to change these when we get a chance.

Reply:
I agree! I made the changes for this.

Reply:
I agree! I made changes for this.